### ISSUE TITLE
Issue with Dropdown "Tutorials"

### ISSUE THREAD

**Issue by kanhegaonkarsaurabh** [ListeningToWaves/musiscope-v2#59]
Currently, if you open the dropdown and click anywhere on the top navbar, the page redirects to www.listeningtowaves.com/tutorials.

I'm guessing the issue is that the dropdown is detecting a click on the whole of the top navbar instead of just the dropdown. 

Video to reproduce the issue: https://www.loom.com/share/40c3ee87cb7f4ab9966a290cee5b7041

**Comment by kanhegaonkarsaurabh** [comment]
This exact issue is also present in the signal-generator and the spectrogram

**Comment by jfong02** [comment]
One easy, though somewhat hacky fix is to include another option in the dropdown menu that does not have a text to it, then put it on the top of the options list. This ensures that this is the first and default value of the dropdown. Since the value of this option is none of the other options, the window will not redirect.

However, the appearance of the dropdown will now include an extra highlighted option as the default selected option. It does not have any text on it, which makes the bar slightly smaller in size.

For example for the Musiscope, the following code can be added to the top of the options list.
`{ key: 'Musiscope', text: '', value: 'Musiscope'},`

**Comment by kanhegaonkarsaurabh** [comment]
Yeah I think this can work @jfong02 . You wanna implement it and show it once to Victor so that he knows what the bug is and how the fix will look like that appearance will have that extra highlighted part
